cockatrice/src/remoteclient.cpp

```diff
@@ -365,7 +367,9 @@ void RemoteClient::disconnectFromServer()
 QString RemoteClient::getSrvClientID(const QString _hostname)
 {
     QString srvClientID = settingsCache->getClientID();
-    srvClientID += _hostname;
+    QHostInfo hostInfo = QHostInfo::fromName(_hostname);
+    QHostAddress hostAddress = hostInfo.addresses().first();
```

Comment:
This block of code isn't even reached if the connection to the server fails.  The connection is made and then the doLogin code fires.  Are you saying that we should check for a lookup error on something that has already been looked up and successfully resolved?

> On Jul 16, 2016, at 5:46 PM, Gavin Bisesi <notifications@github.com> wrote:
> 
> In cockatrice/src/remoteclient.cpp:
> 
> > @@ -365,7 +367,9 @@ void RemoteClient::disconnectFromServer()
> >  QString RemoteClient::getSrvClientID(const QString _hostname)
> >  {
> >      QString srvClientID = settingsCache->getClientID();
> > -    srvClientID += _hostname;
> > +    QHostInfo hostInfo = QHostInfo::fromName(_hostname);
> > +    QHostAddress hostAddress = hostInfo.addresses().first();
> well, we can check QHostInfo::error() and QHostInfo::errorString(). If there's an error on this step we probably can't connect to the server. I think it shouldn't attempt to connect if this fails.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>